style.css

```diff
@@ -45,6 +45,10 @@ body {
   transition: all 0.2s ease;
 }
```

Comment:
- [ ] For the mobile version, please consider fixing the font weight since in the Figma design is a little bolder than the one in your current design 👍🏼 
- [ ] Also the content should take up most of the space as shown in the Figma design 💪🏼 

## Your Design
![image](https://user-images.githubusercontent.com/68030297/228998160-333c4881-0ea4-4484-a611-0e936789b950.png)


## Figma Design
![image](https://user-images.githubusercontent.com/68030297/228998170-1b125aa1-fad3-4eb9-bc28-0de75e60de20.png)

## Suggestions

- A good start will be avoiding setting fixed width to both `title1` and `title2`,  in the CSS both classes has a width of `310px` and `327px`.

Reply:
I used All the properties and typography in the Figma design.